src/Process-Data/Get_Odds_Data.py

```diff
@@ -0,0 +1,66 @@
+import random
+import time
+import pandas as pd
+import sqlite3
+
+from datetime import datetime
+from tqdm import tqdm
+from sbrscrape import Scoreboard
+
+year = [2022, 2023]
+season = ["2022-23"]
+
+month = [10, 11, 12, 1, 2, 3, 4, 5, 6]
+days = [1, 2, 3, 4, 5, 6, 7, 8, 9, 10, 11, 12, 13, 14, 15, 16, 17, 18, 19, 20, 21, 22, 23, 24, 25, 26, 27, 28, 29, 30, 31]
+
+begin_year_pointer = year[0]
+end_year_pointer = year[0]
+count = 0
+year_count = 0
+
+sportsbook='fanduel'
+df_data = []
+
```

Comment:
Any pros / cons to moving to this for odds? I always grabbed them from here
[https://www.sportsbookreviewsonline.com/scoresoddsarchives/nba/nbaoddsarchives.htm](https://www.sportsbookreviewsonline.com/scoresoddsarchives/nba/nbaoddsarchives.htm)

Reply:
> Any pros / cons to moving to this for odds? I always grabbed them from here https://www.sportsbookreviewsonline.com/scoresoddsarchives/nba/nbaoddsarchives.htm

I don't know if it is relevant but the website you linked no longer provides odds after January 16th, 2023. If you need odds for the 2022-23 season, using the sbrscrape library is the only option.

Reply:
They will update it soon. I think they do it in batches.

Reply:
Upon further review you are right. I see their note.